generators/server/templates/entity/src/main/java/package/domain/Entity.java.jhi.jakarta_persistence.ejs

```diff
@@ -77,14 +77,14 @@ import org.hibernate.type.SqlTypes;
 <%_ for (const relationship of relationships) { -%>
 <&_ if (fragment.relationship<%- relationship.relationshipNameCapitalized %>AnnotationSection) { -&>
   <%_ if (relationship.relationshipOneToMany) { -%>
-    @OneToMany(mappedBy = "<%= relationship.otherEntityRelationshipName %>")
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "<%= relationship.otherEntityRelationshipName %>")
```

Comment:
Arent't they lazy by default?

Reply:
@Tcharl I also thought, but see https://github.com/jhipster/generator-jhipster/pull/19791#issuecomment-1311988418 and related discussion.

Reply:
Weird... Maybe mentionning to remove the statement after hibernate 6 could be a plus